### ISSUE TITLE
Req: Move the blacklist of frequent use passwords to a seperate file

### ISSUE THREAD

**Issue by VincentNikkelen** [antelle/passfield#24]
Hi,

I'd like to propose moving the blacklist to a separate file instead of having it included inside the main code.

Each country/language has its own frequent list and this allows users to adjust the blacklist to their needs without the need to edit the main code.

If you agree with this proposal, I could implement it.

Vincent.

Note, one could even consider support for multiple external files so that users can dynamically decide which list to load, but that is not my main need. (Example: assume I am running a soccer website. For Dutch supporters I would load a list with famous Dutch clubs, for English visitors I load the list with English clubs.)

**Comment by antelle** [comment]
Hi! Good proposal, thanks!
I think, there should be one list for English words which can be extended with language-specific dictionaries for different languages. And yes, it's better to store language-specific lists in locales.
I agree, as there are already quite many locales now, I'll refactor locales file and split it to separate files per locale. Now I suggest you to add a word dictionary variable to locales file, that would be much easier and will not require grunt build tuning.

**Comment by VincentNikkelen** [comment]
Just to make sure I have the right understanding:
- Make a separate common blacklist file for common passwords and most used English words
- Add a blacklist variable to locale files. This blacklist is then merged with the common blacklist file

This allows a user to extend localized blacklist files AND allows the user to extend the common blacklist with specific passwords applicable for his purpose (e.g. local sport clubs). This would especially useful for applications targeting a specific country where multiple languages are spoken.

**Comment by antelle** [comment]
For now, I suppose just to add blacklist variable to locale files.
To extend the default black-list, there's `blacklist` init option, items from which will be concatenated with words from default English list.

**Comment by VincentNikkelen** [comment]
Do I understand it right you propose to let the blacklist in the locale files replace the default black-list.
That would create quite some duplication for passwords like '12345', '123456', 'abcdef', 'qwerty', 'azerty' etc.

**Comment by antelle** [comment]
No, they should extend it, like it's done with blacklist from options <a href="https://github.com/antelle/passfield/blob/master/js/passfield.js#L195">here</a>. So lists will be merged and there will be no duplicates.

**Comment by VincentNikkelen** [comment]
Ok.
sorry for the many questions, just trying to avoid miss-understandings. 
I'll wait with implementing until the locates are refactored. (Rather busy anyway rest of the week)

**Comment by antelle** [comment]
So far, I have implemented extending blacklist from locales in 910cc20. Now blacklist is concatenated from: 1. culture-neutral list (English); 2. `options.blackList` provided by user; 3. `blackList` from used locale.
